### PR TITLE
Prevent emails being sent outside production

### PR DESCRIPTION
**Pull request by tatyree**
It should not be sending emails to the prisons from preprod, test or
dev.  I missed this when merging prison_data_production and
prison_data_staging.  This fixes it programatically and for all prisons.

### CODE REVIEW

File: config/initializers/prisons.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Rails.configuration.prisons = []
 
 Dir["config/prisons/*.yml"].each { |file|
-  Prison.create(YAML.load_file(file))
+  prison = YAML.load_file(file)
+  unless Rails.env.production?
+    prison['email'] = "pvb.#{prison['nomis_id']}@maildrop.dsd.io"
```
Comment (SteveMarshall) [inline config/initializers/prisons.rb]: I'm not sure this is correct, is it? For example, it's `pvb.cardiff@maildrop.dsd.io`, if memory serves…

Comment (tatyree) [inline config/initializers/prisons.rb]: It was line of least resistance.  Using names means having to parameterize...Can change it once I've got a Site association in place.  This was just to unblock @threedaymonk on testing on preprod.

Comment (tatyree) [inline config/initializers/prisons.rb]: And, also, not all of the prison_data_staging prisons from the old format had maildrop.dsd.io emails...

Comment (SteveMarshall) [inline config/initializers/prisons.rb]: Fair enough! Go ahead then. As for not having maildrop emails everywhere… :sob:

**Pull request merged**